knowledge-base/kb-panelbar-remove-mouseover-and-selected-overlay-child-item.md

```diff
@@ -0,0 +1,98 @@
+	</tr>
+</table>
+
+## Description
+
+How to remove mouse over and selected background colors of a child RadPanelBarItem when it is hovered or selected.
+
+## Solution
+
+Extract and modify the content of the Template property of RadPanelBarItem.
+
+__To remove the mouse over overlay:__
+
+1. Extract the Style of the RadPanelBarItem. 
+2. Find the Setter that sets the Template property.
+3. Find the VisualState with x:Name="MouseOver" in the template and remove it.
+4. Find the Border element with x:Name="MouseOverVisual" and remove it as well.
+
+The following elements are the ones that should be removed from the template:
+
+#### __[XAML]__
+{{region kb-panelbar-remove-mouseover-and-selected-overlay-child-item-0}}
+	<VisualState x:Name="MouseOver">
+		<Storyboard>
+			<DoubleAnimation Duration="0:0:0.2" To="1" Storyboard.TargetProperty="Opacity" Storyboard.TargetName="MouseOverVisual"/>
+		</Storyboard>
+	</VisualState>
+	<!---------------------------->
+	<Border x:Name="MouseOverVisual" BorderBrush="#FFFFC92B" BorderThickness="1" CornerRadius="1" Opacity="0">
+		<Border.Background>
+			<LinearGradientBrush EndPoint="0.5,1" StartPoint="0.5,0">
+				<GradientStop Color="#FFFFFBA3" Offset="1"/>
+				<GradientStop Color="#FFFFFBDA" Offset="0"/>
+			</LinearGradientBrush>
+		</Border.Background>
+		<Border BorderBrush="White" BorderThickness="1" Background="{x:Null}" CornerRadius="0"/>
+	</Border>
+{{endregion}}
+
+__To remove the selected overlay:__
+
+1. Extract the Style of the RadPanelBarItem. 
+2. Find the Setter that sets the Template property.
+3. Find the VisualState with x:Name="MouseOver" in the template and remove it.
+4. Find the Border element with x:Name="MouseOverVisual" and remove it as well.
+
+The following elements are the ones that should be removed from the template:
+
+#### __[XAML]__
+{{region kb-panelbar-remove-mouseover-and-selected-overlay-child-item-1}}
+	<VisualState x:Name="Selected">
+		<Storyboard>
+			<DoubleAnimation Duration="0" To="1" Storyboard.TargetProperty="Opacity" Storyboard.TargetName="SelectedVisual"/>
+		</Storyboard>
+	</VisualState>
+	<!---------------------------->
+	<Border x:Name="SelectedVisual" BorderBrush="#FFFFC92B" BorderThickness="1" CornerRadius="1" Opacity="0">
+		<Border.Background>
+			<LinearGradientBrush EndPoint="0.5,1" StartPoint="0.5,0">
+				<GradientStop Color="#FFFCE79F" Offset="1"/>
+				<GradientStop Color="#FFFDD3A8"/>
+			</LinearGradientBrush>
+		</Border.Background>
+		<Border BorderBrush="White" BorderThickness="1" Background="{x:Null}" CornerRadius="0"/>
+	</Border>
+{{endregion}}
+
+You can apply the custom style using an implicit style. This is a Style without its x:Key set.
+
+#### __[XAML]__
+{{region kb-panelbar-remove-mouseover-and-selected-overlay-child-item-2}}
+	<Style TargetType="telerik:RadPanelBarItem">
+		<!-- Some setters here -->
+	</Style>
+{{endregion}}
+
+Or you can apply the style via the __ItemContainerStyle__ property of the RadPanelBar control.
```

Comment:
We can leave a See Also link to the Editing Control Templates article as it might be useful. Alternatively, we can also mention the article at the beginning.